live-build/misc/ansible-roles/appliance-build.virtualization-common/tasks/main.yml

```diff
@@ -23,29 +23,6 @@
     path: /var/opt/delphix
     state: directory
 
-#
-# The "product", "platform", and "cr_auth" fields are intentionally left
-# out of this file, as those fields should get filled in by other roles
-# that will run after this role.
-#
-- copy:
-    dest: /var/opt/delphix/server.conf
```

Comment:
Do we want to also cleanup this file if it already exists to avoid confusing people in the future? We can do this in another PR though.